src/web/components/SeriesSectionLink.tsx

```diff
@@ -43,8 +44,9 @@ const primaryStyle = css`
 	${from.leftCol} {
 		${headline.xxsmall({ fontWeight: 'bold' })};
 	}
-
-	padding-right: ${space[2]}px;
+	${until.leftCol} {
+		margin-right: ${space[2]}px;
+	}
```

Comment:
This spacing is much more conspicuous when there's a background on the series title but not the section title, hence margin, and hence why we only want it if these two things are on one row